### PR TITLE
replace config-model hack with ngxs approach

### LINKED ISSUE #1092: Rewrite config model with NGXS

### PR DESCRIPTION
**Pull request by Epsilon02**
close #1092

### CODE REVIEW

File: frontend/src/app/@core/services/config/web/config.service.ts [hunk 1]
```diff
@@ -13,17 +13,12 @@ export class ConfigWebService implements IConfigService {
...
 
-  getConfig(): Observable<ConfigModel> {
-    const config = localStorage.getItem('config');
-    if (config) {
-      return of(new ConfigModel(JSON.parse(config).global));
-    } else {
-      return of(new ConfigModel());
-    }
+  loadConfig(): Observable<Config> {
+    throw Error('Loadconfig is for web not required');
```
Comment (crapStone) [inline frontend/src/app/@core/services/config/web/config.service.ts:17]: ... not required for web

File: frontend/src/app/@core/services/config/web/config.service.ts [hunk 2]
```diff
@@ -13,17 +13,12 @@ export class ConfigWebService implements IConfigService {
...
   }
 
-  saveConfig(config: ConfigModel): Observable<void> {
-    return of(localStorage.setItem('config', JSON.stringify(config)));
+  saveConfig(_: any): Observable<void> {
+    throw Error('Saveconfig is for web not required');
```
Comment (crapStone) [inline frontend/src/app/@core/services/config/web/config.service.ts:21]: here too

**Review by crapStone — CHANGES_REQUESTED** [review]
The settings dialog state is not consistent, but I have to investigate more.

**Comment by Epsilon02** [comment]
> The settings dialog state is not consistent, but I have to investigate more.

Hm please try it in a incognito window maybe there is a problem with the localstorage :thinking: 
I'm not sure I had this issue too, because I cleared my localstorage during testing.

**Review by crapStone — APPROVED** [review]

**Pull request merged**